README.md

```diff
@@ -1,57 +1,91 @@
+    > **Windows Users** - You can install Anaconda directly to your Windows box OR you can run via a linux
+    environment using the Windows Subsystem for Linux (WSL).  See
+    [this tutorial on WSL](https://tsdat.readthedocs.io/en/latest/tutorials/wsl.html) for
+    how to set up a WSL environment and attach VS Code to it.
 
-> **Windows Users** - we strongly recommend developing in a linux environment to help
-mitigate common python packaging and path issues. See
-[this tutorial on WSL](https://tsdat.readthedocs.io/en/latest/tutorials/wsl.html) for
-how to set up a WSL environment and attach VS Code to it.
 
-1. Click '[Use this template](https://github.com/tsdat/pipeline-template/generate)' and
-follow the steps to copy this into to your account.
-    > If you are looking to get an older version of the template, you will need to
+## Creating a repository from the pipeline-template
+You can create a new repository based upon the tsdat pipeline-template repository in GitHub:
+
+1. Click this '[Use this template](https://github.com/tsdat/pipeline-template/generate)' link and
+follow the steps to copy the template repository into to your account.
+    > **NOTE:** If you are looking to get an older version of the template, you will need to
     select the box next to 'Include all branches' and set the branch your are interested
     in as your new default branch.
 
-2. On github click the 'Code' button to get a link to your code, then run `git clone <the link you copied>` from the terminal on your computer where you would like to work on the code.
+2. On github click the 'Code' button to get a link to your code, then run 
+    ```bash
+    git clone <the link you copied>
+    ```
+    from the terminal on your computer where you would like to work on the code.
+
+## Setting up your Anaconda environment
+1. Open an appropriate terminal shell from your computer
+   1. If you are on Linux or Mac, just open a regular terminal
+   2. If you are on Windows, start your Anaconda prompt if you installed Anaconda directly to Windows, OR open a
+   WSL terminal if you installed Anaconda via WSL.
 
-3. Open the cloned repository in VS Code. This repository contains default settings for
-VS Code that will make it much easier to get started quickly.
 
-4. Install the recommended extensions (there should be a pop-up in VS Code with recommendations).
+2. Run the following commands to create and activate your conda environment, where $REPOSITORY_ROOT represents
+the folder where you checked out your pipeline repository:
 
-5. We highly recommend using [conda](https://docs.anaconda.com/anaconda/install/) to
-manage dependencies in your development environment. 
     ```bash
-    $ conda create --name tsdat-pipelines python=3.8
-    $ conda activate tsdat-pipelines
-    (tsdat-pipelines) $ pip install -r requirements-dev.txt
+    conda env create --name tsdat-pipelines --file=conda-environment.yaml
```

Comment:
The `--name tsdat-pipelines` part isn't needed here since it's specified in the `conda-environment.yaml` file